examples/vision/md/image_captioning.md

```diff
@@ -677,30 +800,36 @@ generate_caption()
 ```
 
 
+    
 ![png](/img/examples/vision/image_captioning/image_captioning_17_0.png)
+    
 
 
 <div class="k-default-codeblock">
 ```
-PREDICTED CAPTION: a man is jumping over a horse
+Predicted Caption:  a group of dogs race in the snow
```

Comment:
The resulting captions on the test images don't look accurate at all, is that expected? Should we just train the model for longer? If that's the case, then I suggest showing the *final* results (in a text block) rather than displaying results from the underfit model.

Reply:
@fchollet The test image is different here. This caption corresponds to this [new test image](https://github.com/AakashKumarNain/keras-io/blob/51cb6c626fefe757b30a2bb632cbcb6a100a8db1/examples/vision/img/image_captioning_17_0.png). Can you please double check again?

Reply:
And the reason for test image being different is that I have added another check for filtering the data

Reply:
I don't know how is this possible but the images in this PR didn't get reflected in the master branch after merging. The test images in this PR are different from the images used in the first version of the code example